### PR TITLE
pdctl: add region and store command

### PR DESCRIPTION
**Pull request by nolouch**
PTAL @huachaohuang @siddontang @overvenus

### CODE REVIEW

File: pdctl/command/store_command.go [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	cli := &http.Client{}
+	if _, err := strconv.Atoi(args[0]); err != nil {
+		fmt.Println("store_id should be a number")
+		return
+	}
+	prefix := fmt.Sprintf(storePrefix, args[0])
+	url := getAddressFromCmd(cmd, prefix)
+	r, err := http.NewRequest("DELETE", url, nil)
+	if err != nil {
+		fmt.Printf("Failed to delete store %s: [%s]\n", args[0], err)
```
Comment (huachaohuang) [inline pdctl/command/store_command.go]: return

**Comment by huachaohuang** [comment]
We can support showing stores with different state (up,offline,tombstone) later.
Rest LGTM.

File: cmd/pd-ctl/main.go [hunk 2]
```diff
@@ -23,12 +22,17 @@ import (
...
 
-var url string
+var (
+	url    string
+	detach bool
+)
 
 func init() {
-	flag.StringVar(&url, "u", "http://127.0.0.1:2379", "the pd address")
+	flag.StringVarP(&url, "pd_url", "u", "http://127.0.0.1:2379", "the pd address")
```
Comment (siddontang) [inline cmd/pd-ctl/main.go]: s/pd_url/pd/

File: pdctl/command/store_command.go [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+var (
+	storesPrefix = "pd/api/v1/stores"
+	storePrefix  = "pd/api/v1/store/%s"
+)
+
+// NewStoreCommand return a store subcommand of rootCmd
+func NewStoreCommand() *cobra.Command {
+	s := &cobra.Command{
+		Use:   "store [delete] <store_id>",
+		Short: "show the sotre status",
```
Comment (siddontang) [inline pdctl/command/store_command.go]: store

**Comment by siddontang** [comment]
Can you give some example on how to use it?

**Comment by GregoryIan** [comment]
how about use a global config like kubectl in detach mode，so that we don't need to input url ...

**Comment by nolouch** [comment]
@GregoryIan  good idea

File: pdctl/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+1. Make sure [*Go*](https://golang.org/) (version 1.5+) is installed.
+2. Use `make` in pd root path. `pdctl` will build in `bin` directory.
+
+## Usage
+
+### Example
+run:
+    
+    ./pdctl store -d  -u 127.0.0.1:2379
+show all stores status. '-u' specify the pd address, it can be overwritten by envirment variours like `export PD_ADDR=127.0.0.1:2379`
```
Comment (siddontang) [inline pdctl/README.md]: environment variables

File: pdctl/command/store_command.go [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+func NewDeleteStoreCommand() *cobra.Command {
+	d := &cobra.Command{
+		Use:   "delete <store_id>",
+		Short: "delete the store",
+		Run:   deleteStoreCommandF,
+	}
+	return d
+}
+
+func showStoreCommandF(cmd *cobra.Command, args []string) {
```
Comment (siddontang) [inline pdctl/command/store_command.go]: what's `F` here for?

**Comment by nolouch** [comment]
@siddontang  PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**